samcli/local/common/runtime_template.py

```diff
@@ -86,6 +86,7 @@ def get_local_lambda_images_location(mapping, runtime):
     "ruby2.7": ["bundler"],
     "nodejs12.x": ["npm"],
     "nodejs10.x": ["npm"],
+    "dotnet5.0": ["cli-package"],
```

Comment:
Not needed.

Reply:
I think, this is tied to L41 above. Both changes are required to `sam init` the template.